entrypoint.sh

```diff
@@ -52,14 +52,14 @@ elif echo "${LABELS}" | grep "bump:minor" ; then
 elif echo "${LABELS}" | grep "bump:patch" ; then
   BUMP_LEVEL="patch"
 fi
-echo "::set-output name=level::#{BUMP_LEVEL}"
+echo "level=#{BUMP_LEVEL}" >> $GITHUB_OUTPUT
```

Comment:
is dit veranderd in GitHub maar doet het eigenlijk hetzelfde? Of is er iets anders aan de hand?

Reply:
Ja is veranderd. Momenteel zie je veel van het volgende in de actions:
<img width="1018" alt="Screenshot 2023-03-06 at 14 36 33" src="https://user-images.githubusercontent.com/331876/223125379-17da0daf-c68d-41e6-b86f-f895f4a0d90d.png">

zie ook: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/